osmosis/assetlist.json

```diff
@@ -1316,6 +1316,49 @@
       "logo_URIs": {
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/injective/images/autism.png"
       }
+    },
+    {
+      "description": "Woof on Osmosis",
+      "denom_units": [
+        {
+          "denom": "ibc/9B8EC667B6DF55387DC0F3ACC4F187DA6921B0806ED35DE6B04DE96F5AB81F53",
+          "exponent": 0,
+          "aliases": [
+            "uwoof"
+          ]
+        },
+        {
+          "denom": "WOOF",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/9B8EC667B6DF55387DC0F3ACC4F187DA6921B0806ED35DE6B04DE96F5AB81F53",
+      "name": "WOOF chihuahua",
+      "display": "woof",
```

Comment:
this display denom is not defined within denom_units. Should be 'WOOF'?

Reply:
True! Fixed on this commit https://github.com/cosmos/chain-registry/pull/3545/commits/e6a428aa3cccb2826404068ec18b606218cb0ac9
Thanks